mitosheet/mitosheet/utils.py

```diff
@@ -82,7 +82,61 @@ def is_default_df_names(df_names: List[str]) -> bool:
     """
     return len(df_names) > 0 and df_names == [f'df{i + 1}' for i in range(len(df_names))]
 
+
+def get_conditonal_formatting_result(
+        state: StateType,
+        sheet_index: int,
+        df: pd.DataFrame,
+        conditional_formatting_rules: List[Dict[str, Any]],
+        max_rows: Optional[int]=MAX_ROWS,
+    ) -> ConditionalFormattingResult: 
+
+    df = df.head(max_rows)
+
+    invalid_conditional_formats: ConditionalFormattingInvalidResults = dict()
+    formatted_result: ConditionalFormattingCellResults = dict()
+
+    for conditional_format in conditional_formatting_rules:
+        format_uuid = conditional_format["format_uuid"]
+        column_ids = conditional_format["columnIDs"]
+
+        for column_id in column_ids:
+            # Skip columns this is not applied to
+            if column_id not in column_ids:
+                continue
```

Comment:
I don't think we need this check. We're iterating over colum_id in column_ids already